docs/sources/mimir/release-notes/v2.12.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "Grafana Mimir version 2.12 release notes"
+menuTitle: "V2.12 release notes"
+description: "Release notes for Grafana Mimir version 2.12"
+weight: 1000
+---
+
+# Grafana Mimir version 2.12 release notes
+
+<!-- vale Grafana.GoogleWill = NO -->
+<!-- vale Grafana.Timeless = NO -->
+<!-- Release notes are often future focused -->
+
+Grafana Labs is excited to announce version 2.12 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bug fixes in this release.
+For the complete list of changes, refer to the [CHANGELOG](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Added `mimir-distroless` container image, built upon the `distroless` image**.
```

Comment:
"the distroless image" isn't quite right as there are a number.

Built upon "a" distroless image perhaps. The specific one at the moment is `gcr.io/distroless/static-debian12`

Reply:
Thank you @jhesketh.
I have updated the PR. Does this look better?